docs/changelog/109414.yaml

```diff
@@ -0,0 +1,6 @@
+pr: 109414
+summary: Don't trip circuit breakers for retention lease sync actions
```

Comment:
I've broadened the PR title given the additional requirement around indexing pressure checks. If we're happy with that I'll update the changelog to be the same.